dependencies/README.md

```diff
@@ -72,7 +72,7 @@ and annotates linkage errors with [reachability](
 ../library-best-practices/glossary.md#reachability) from [entry point classes](
 ../library-best-practices/glossary.md#entry-point-class).
 
-Entry point classes are different for the targets of checks:
-  - **Check for a BOM**: classes in the Maven artifacts listed in the BOM
-  - **Check for a project**: classes in the direct dependencies of the project
-  - **Check for a list of Maven artifacts**: classes in the Maven artifacts
+Entry point classes are different for the input of checks:
+  - **Check for a Maven BOM**: classes in the Maven artifacts listed in the BOM
+  - **Check for a list of Maven coordinates**: classes in the Maven artifacts
+  - **Check for a list of class and jar files**: classes in the list and the jar files
```

Comment:
`all classes in the input are entry points. This means that every linkage error is considered reachable.`

Reply:
Updated.